lib/jsonapi/consumer/helpers/dynamic_attributes.rb

```diff
@@ -7,7 +7,7 @@ def attributes
       end
 
       def attributes=(attrs = {})
-        @attributes ||= ActiveSupport::HashWithIndifferentAccess.new
+        @attributes ||= ActiveSupport::HashWithIndifferentAccess.new(attrs)
 
         return @attributes unless attrs.present?
         attrs.each do |key, value|
```

Comment:
This should be run, no?

Reply:
Yes. 

Needed for non-activerecord objects